examples/extract_data/src/extract_data_login.py

```diff
@@ -0,0 +1,146 @@
+        if profile_id:
+            print(f"Using profileId: {profile_id}")
+        else:
+            print("No profileId provided")
+            profile_id = None
+
+        print("Creating sessions")
+        configuration = SessionConfigV1(
+            timeout_minutes=10,
+            persist_profile=not profile_id,
+            base_profile_id=profile_id
+        )
+        session = client.sessions.create(configuration=configuration)
+
+        if not session.data.cdp_ws_url:
+            raise ValueError("Unable to get cdp url")
+
+
+        session_id = session.data.id if session.data else None
+        print("Created airtop session", session_id)
+
+        # Create a new window and navigate to the URL
+        window = client.windows.create(session_id, url=LOGIN_URL)
+        window_info = client.windows.get_window_info(session_id, window.data.window_id)
+
+        # Check whether the user is logged in
+        print("Determining whether the user is logged in...")
+        logged_in_schema = IS_LOGGED_IN_OUTPUT_SCHEMA
+        is_logged_in_response = client.windows.page_query(
+            session_id, window.data.window_id, prompt=IS_LOGGED_IN_PROMPT, configuration=PageQueryConfig(
+                output_schema=logged_in_schema
+            )
+        )
+        print("Response if user is logged in")
+        parsed_response = json.loads(is_logged_in_response.data.model_response)
+        if "error" in parsed_response:
+            raise ValueError(parsed_response.error)
+        is_user_logged_in = parsed_response["isLoggedIn"]
+
+        # Prompt the user to log in if not already logged in
+        if not is_user_logged_in:
+            print(f"Log into your Glassdoor account. Press Enter once done. Live view URL: {window_info.data.live_view_url}")
+            input()
+            print(f"To avoid logging in again, use the profileId next time: {session.data.profile_id}")
+        else:
+            print(f"User is already logged in. Live view URL: {window_info.data.live_view_url}")
+
+        # Navigate to the target URL
+        print("Navigating to target URL...")
+        client.windows.load_url(session_id, window.data.window_id, url=TARGET_URL)
+        print("Prompting the AI agent, waiting for a response...")
+
+        extract_data_response = client.windows.page_query(
+            session_id, window.data.window_id,
+                prompt=EXTRACT_DATA_PROMPT,
+                configuration=PageQueryConfig(
+                output_schema=EXTRACT_DATA_OUTPUT_SCHEMA
+            )
+        )
+        formatted_json = json.dumps(json.loads(extract_data_response.data.model_response), indent=2)
+        print("Response:\n\n", formatted_json)
+
+    except Exception as e:
+        try:
+            print(e.status_code, e.message, e.body)
+        except:
+            print(e)
+    
+    finally:
+        # Clean up
+        if client is not None and session_id is not None:
```

Comment:
This should be
```
        # Clean up
        if client is not None and session_id is not None:
            if window is not None and window.data is not None and window.data.window_id is not None:
                client.windows.close(session_id, window.data.window_id)
            client.sessions.terminate(session_id)
            print("Session terminated")
```

Otherwise close could fail 
![image](https://github.com/user-attachments/assets/76921bad-3b4b-4cf2-8a1d-61c14abff5e1)